pkg/controller/daemon/update.go

```diff
@@ -548,7 +548,7 @@ func (dsc *DaemonSetsController) updatedDesiredNodeCounts(ds *apps.DaemonSet, no
 		klog.Warningf("DaemonSet %s/%s is not configured for surge or unavailability, defaulting to accepting unavailability", ds.Namespace, ds.Name)
```

Comment:
can you modify the rest log structure of these files as well? thanks!
For example L548.

Reply:
Fixed. I couldn't find any others though.